lib/richmond/rtfm.rb

```diff
@@ -104,15 +106,20 @@ def set_output_file?(line)
     end
 
     def assert_directory_exists(filename)
-      dirname = File.dirname(filename)
-      unless File.directory?(dirname)
-        FileUtils.mkdir_p(dirname)
-      end
+      #dirname = File.dirname(filename)
+      #unless File.directory?(dirname)
+        #FileUtils.mkdir_p(dirname)
+      #end
```

Comment:
I added this code due to an actual bug I encountered--but you're right I didn't test it. :)